media/redesign/js/wiki.js

```diff
@@ -578,6 +578,17 @@
         }
     });
 
+    /*
+      Make Compare selected revisions button sticky when scrolling history page
+    */
+    (function() {
+        var $revisionButtonOffset = $('.revision-list-controls .link-btn').offset().top;
```

Comment:
You use `$('.revision-list-controls .link-btn')` twice;  I would create a reference to that button so you aren't constantly retrieving it:

```
var button = $('.revision-list-controls .link-btn');
```